backend/model/movies-model.js

```diff
@@ -0,0 +1,13 @@
+// Load mongoose package
+import mongoose from 'mongoose';
+
+// Create a schema
+const MovieSchema = new mongoose.Schema({
```

Comment:
I think that if we have another schema, we'll have another "model" file.
But as far as I understand, it's not a problem to do `mongoose.Schema` in all "models" files.